.vscode/settings.json

```diff
@@ -16,19 +16,4 @@
       "url": "https://json.schemastore.org/chrome-manifest.json"
     }
   ],
-  "cSpell.words": [
-    "accountsstage",
-    "adduser",
-    "arethetypeswrong",
-    "attw",
-    "BAPI",
-    "citty",
-    "execa",
-    "FAPI",
-    "jwks",
-    "lclclerk",
-    "lclstage",
-    "nextjs",
-    "publint"
-  ],
```

Comment:
❓ Why did we remove those? All those words are part of either a dependency we are using or a Clerk-related keyword.

Reply:
As noted in the PR description, we don't use https://cspell.org/ so no reason to have it in there